CONTRIBUTING.md

```diff
@@ -0,0 +1,68 @@
+# Contributing to ideasbox.lan
+
+Do you like to make the ideasbox.lan project better? Welcome! This document
+will try to provide guidelines about what and how you can help.
+
+
+## Contact and resources
+
+If you have any question, problem or idea, feel free to get in touch with
+project's maintainers:
+
+* Chat using #ideasbox channel on Freenode. If you are not familiar with
+  [IRC](https://en.wikipedia.org/wiki/Internet_Relay_Chat), you can try
+  this simple [online chat application](https://kiwiirc.com/client/irc.freenode.net/?nick=new-user|?#ideasbox):
+  just click "Start" then chat!
+
+* Ask questions, report issues or propose ideas as
+  [tickets] (https://github.com/issues).
+
+Here are online resources you may find useful to contribute to the project:
+
+* Homepage: http://www.ideas-box.org/
+* Code repository: https://github.com/ideas-box/ideasbox.lan
+* Issues, questions and feature requests:
+  https://github.com/ideas-box/ideasbox.lan/issues
+* Continuous integration: https://travis-ci.org/ideas-box/ideasbox.lan
+* Documentation:
+
+  * docs: https://github.com/ideas-box/ideasbox.lan/blob/master/docs/index.md
+  * wiki: https://github.com/ideas-box/ideasbox.lan/wiki
```

Comment:
Wiki should be community driven, without merge/commit/review workflow, more accessible.

But actually that was the first attempt of doc I had a bit in a rush. I think we can omit this link for now, there is no more valuable info there.